docs/source/deployment/databricks.md

```diff
@@ -263,3 +263,9 @@ Kedro-Viz can then be launched in a new browser tab with the `%run_viz` line mag
 ```ipython
 In [2]: %run_viz
 ```
+
+### How to use Datasets stored on Databricks DBFS
+
+DBFS is a distributed file system mounted into a DataBricks workspace and accessible on a DataBricks cluster. It maps cloud object storage URIs to relative paths so as to simplify the process of persisting files. With DBFS, libraries can read from or write to distributed storage as if it's a local file.
+To use datasets like `SparkDataSet` with DBFS, the file path must be prefixed with `/dbfs`.
+> **Note**: Datasets that use the local filesystem, such as Kedro's Pandas datasets, will return a `DataSetError` if the code tries to load files from a remote DBFS save location. Do not use the `/dbfs` prefix for these datasets.
```

Comment:
At the moment, I don't think this is accurate. All Python code (apart from PySpark) will try to resolve a file path in driver node storage by default, but it can resolve file paths on DBFS if the file path points to it. Therefore, adding the `/dbfs` prefix is not only valid but sometimes what the user needs to do.

The rules for whether code will look in DBFS or in driver node storage by default are described [here](https://docs.databricks.com/files/index.html). We should link to this documentation in this section also.